docs/installation/linux.md

```diff
@@ -352,9 +379,14 @@ sudo installer -pkg powershell-6.0.0-beta.1-osx.10.12-x64.pkg -target /
 
 ### Uninstallation - macOS 10.12
 
-PowerShell on MacOS must be removed manually.
+If you PowerShell with Homebrew, uninstallation is easy:
```

Comment:
Maybe `If you PowerShell installed with Homebrew` ? Not an English expert, but this language feels too playful :)